features/step_definitions/networking.rb

```diff
@@ -725,14 +727,31 @@
   ensure_admin_tagged
   node = node(node_name)
   host = node.host
-  @result = host.exec_admin("/sbin/bridge vlan show")
+  @result = host.exec_admin("/sbin/bridge -j vlan show")
```

Comment:
Cool stuff. I didn't know that

Reply:
The next step is to see if `jq` is installed everywhere.

Looks like some scripts are using it
https://github.com/openshift/machine-config-operator/pull/1860/files#diff-8197d7b27eb62b1ba2b9b44460cd3238R17

```bash
      iface=$(ip -j route show default | jq -r '.[0].dev')
```